src/components/DimensionsPanel/List/DimensionItem.js

```diff
@@ -27,26 +28,16 @@ class DimensionItem extends Component {
     getDimensionIcon = () => {
         const Icon = getPredefinedDimensionProp(this.props.id, 'icon')
         return Icon ? (
-            <Icon style={styles.fixedDimensionIcon} />
+            <Icon className={style.fixedDimensionIcon} />
```

Comment:
this + "dynamic-dimension-icon" below are a little bit strange to me, since I can't find those classes anywhere in style/css file...

Reply:
Same, I didn't really know what it was referring to, but I thought I'd be mindful of [Chesterton's Fence.](https://en.wiktionary.org/wiki/Chesterton%27s_fence)